bin/cron_pull_tweets.py

```diff
@@ -33,17 +35,38 @@ def run_task(params, ntweets=3, maxpages=2):
     cursor = tweepy.Cursor(api.search, q='bitcoin', lang='en', count=ntweets, tweet_mode='extended')
     stored_at = datetime.now()
 
+    tweets = []
     for page in cursor.pages(maxpages):
         for result in page:
             tid = result.id
             es_tweet = es_client.ESTweet.get(tid, ignore=404)
             if not es_tweet:
+                tweets.append(es_client.tweepy_to_dict(result))
                 t = es_client.tweet_to_estweet(result,
                                                stored_at=stored_at,
                                                version=ES_TWEET_VERSION)
-                t.save(index='tweets')  # TODO: explicitly giving the index shouldn't be necessary
-            else:
-                print(es_tweet)
+                t.save(index='tweets')
+
+    # Precompute mean sentiment:
```

Comment:
make Precompute mean sentiment and Precompute weighted score separate functions so that it is easier to modify without side effects if needed in the future.

Reply:
Refactored in [4b3418b](https://github.com/fund3/sentiment-dashboard/pull/6/commits/4b3418b639490ec345ee9ed21e6a995713c2e4b7)